recipes/wolfssl/all/conanfile.py

```diff
@@ -0,0 +1,112 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from contextlib import contextmanager
+import os
+
+
+class WolfSSLConan(ConanFile):
+    name = "wolfssl"
+    license = "GPL-2.0"
```

Comment:
```suggestion
    license = "GPL-2.0-or-later"
```

See e.g. https://github.com/wolfSSL/wolfssl/blob/5376763638a0f0dc4889d4ea4b7f5081db1045a0/src/tls13.c#L10